salt/pillar/search.sls

```diff
@@ -24,6 +24,10 @@ search:
     ttl: 0
     rate_limit_minimum_page: 2
 
+    feature_rds: true
+    {% import_yaml "rds-articles.yaml" as rds_articles %}
+    rds_articles: {{ rds_articles|yaml }}
```

Comment:
Can remove the import now?

Reply:
It reflects what is going on in `builder-configuration` to keep it tested in case of refactoring.